run-tests.py

```diff
@@ -64,20 +65,20 @@ def run_doc_tests():
     used).
     """
     print("Checking if docs can be built.")
-    p = subprocess.Popen(
-        ["sphinx-build", "-b", "doctest", "docs", "docs/build/html", "-W"]
-    )
+    p = subprocess.Popen(["sphinx-build", "-b", "doctest", "docs", "docs/build/html"])
```

Comment:
Can't we pass an --ignore equivalent flag here to ignore testing the notebooks? I'm not sure if something like this exists

Reply:
Ah, just read your comment, will check if there exists a solution.

Reply:
Looks like we have another issue: the doctests now create a `docs/build/` folder containing static files that are not deleted upon completion; re-running the doctests builds a duplicate of the docs with that folder included. This causes an array of several other errors 🙁